packages/dds/tree/src/core/schema-stored/schema.ts

```diff
@@ -155,15 +178,16 @@ export interface ErasedTreeNodeSchemaDataFormat
 	extends ErasedType<"TreeNodeSchemaDataFormat"> {}
 
 function toErasedTreeNodeSchemaDataFormat(
-	data: TreeNodeSchemaDataFormat,
+	data: TreeNodeSchemaDataFormatV1 | TreeNodeSchemaDataFormatV2,
 ): ErasedTreeNodeSchemaDataFormat {
 	return data as unknown as ErasedTreeNodeSchemaDataFormat;
 }
 
+// TODO: Should this always return a V1 Format?
```

Comment:
ErasedTreeNodeSchemaDataFormat only exists to type erase the schema format returned by TreeNodeStoredSchema.encode.

Since we removed the package export for TreeNodeStoredSchema, its reason for existance (and thus this function existing) is gone.

TreeNodeStoredSchema.encode should be made to take in SchemaFormatVersion, return TreeNodeSchemaDataFormatV1 | TreeNodeSchemaDataFormatV2 and ErasedTreeNodeSchemaDataFormat and the related functions to cast to and from it should be removed. Ideally do this cleanup in a separate PR (using just TreeNodeSchemaDataFormat since v2 won't exist yet).